app/controllers/invite_users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class InviteUsersController < ApplicationController
```

Comment:
Usually you name controllers and models in the context of a resource. If you want your friend to join, you don't give them an "invite user", you give them an "invitation". In a RESTful design, you are dealing with resources.